src/js/service/ImportService.js

```diff
@@ -0,0 +1,126 @@
+/* Image and Animation import service supporting the import dialogs,
+ * file dropper service, and headless import API. */
+(function () {
+  var ns = $.namespace('pskl.service');
+
+  /**
+   * @param {!PiskelController} piskelController
+   * @param {!PreviewController} previewController
+   * @constructor
+   */
+  ns.ImportService = function (piskelController, previewController) {
+    this.piskelController_ = piskelController;
+    this.previewController_ = previewController;
+  };
+
+  ns.ImportService.prototype.init = function () {
```

Comment:
Presumably there's some sort of assumption that all services have an init function? Worth a short comment here explaining why we have a noop function?